pom.xml

```diff
@@ -441,6 +441,12 @@
 
     <!-- Maven Repositories -->
     <repositories>
+        <repository>
+            <id>openam-dependencies</id>
+            <name>Forgerock OpenAm dependencies</name>
+            <url>https://maven.forgerock.org/repo/openam-dependencies</url>
```

Comment:
We don't want to depend on FR's repo. This is not an acceptable solution.

Reply:
I have removed the forgerock repo dependency.

Additionally, I've added Github workflows script.